docs/networks/node-ops/access-onchain-data/access-nodes/access-node-setup.md

```diff
@@ -4,6 +4,23 @@ sidebar_label: Access Node Setup
 sidebar_position: 2
 ---
 
+```mdx-code-block
```

Comment:
Trying the example given here: https://docusaurus.io/docs/markdown-features/tabs

Reply:
The source of that example is here: https://github.com/facebook/docusaurus/blob/main/website/docs/guides/markdown-features/markdown-features-tabs.mdx

Reply:
@Aliserag